edk2toolext/environment/extdeptypes/nuget_dependency.py

```diff
@@ -137,6 +141,7 @@ def fetch(self):
         cmd += ["install", package_name]
         cmd += ["-Source", self.source]
         cmd += ["-ExcludeVersion"]
```

Comment:
you can't add the NonInteractive flag otherwise local dev is broken.

Reply:
We're already broken for local dev. Since we're using RunCmd for Nuget resolution we both don't get output and don't have a way to forward input to the process.

Reply:
So we just hang asking for authentication